src/client.js

```diff
@@ -31,7 +31,7 @@ let initialize = function(config, emitter) {
     );
   } catch(e) {
     if (e instanceof TypeError) {
-      emitter.error(`Could not initialize Nexmo SDK. Please run 'nexmo setup' to setup the CLI correctly. (${e.message})`);
+      emitter.error(`Could not initialize Nexmo library. Please run 'nexmo setup' to setup the CLI correctly. (${e.message})`);
```

Comment:
Should probably rename to CLI. User doesnt really need to know this uses a library/SDK under the hood.